src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -358,6 +367,55 @@ const TickDisplay = (() => {
                     counter++;
                 }
             }
+            if (contract_category === 'reset' && data.history) {
+                plotResetSpot();
+            }
+        }
+    };
+
+    const removePlotLine = (id, type = 'y') => {
+        if (!chart) return;
+        chart[(`${type}Axis`)][0].removePlotLine(id);
+    };
+
+    const plotResetSpot = (r_barrier) => {
+        if (reset_spot_plotted || !chart) return;
+
+        const reset_time_index = Math.floor(number_of_ticks / 2);
+        const entry_barrier    = entry_spot * 1;
+        const reset_barrier    = r_barrier || absolute_barrier * 1;
+
+        if (entry_barrier !== reset_barrier) {
+            removePlotLine('tick-barrier', 'y');
+
+            chart.yAxis[0].addPlotLine({
+                id    : 'tick-reset-barrier',
+                value : reset_barrier,
+                label : { text: `Reset Barrier (${addComma(reset_barrier)})`, align: 'center' },
```

Comment:
missing localize? same on below lines for label

Reply:
suggestion: change to `localize('Reset Barrier ([_1])', addComma(reset_barrier))` so it matches the generate-static-data text